src/index.js

```diff
@@ -1,47 +1,18 @@
 // import _ from 'lodash';
 import './style.css';
 
-const listArray = [
-  {
-    description: 'Workout at 11am',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Complete projects',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Revision of Work',
-    completed: true,
-    index: 3,
-  },
-];
+import { addList, displayList } from './modules/UI.js';
+import localGet from './modules/Store.js';
 
-listArray.forEach((listObj) => {
-  const { description, completed, index } = listObj;
-  const ulLists = document.querySelector('.ul-lists');
+const form = document.querySelector('.add-list');
+const todoTask = document.getElementById('input');
 
-  const list = document.createElement('li');
-  list.setAttribute('class', 'li-list');
-  list.setAttribute('id', index);
-  ulLists.appendChild(list);
+window.addEventListener('load', () => {
+  displayList();
+});
 
-  const checkCont = document.createElement('div');
-  checkCont.className = 'checkbox-cont';
-  list.appendChild(checkCont);
-
-  const checkbox = document.createElement('input');
-  checkbox.setAttribute('type', 'checkbox');
-  checkbox.checked = completed;
-  checkCont.appendChild(checkbox);
-
-  const paragraph = document.createElement('p');
-  paragraph.textContent = description;
-  checkCont.appendChild(paragraph);
-
-  const ellipsis = document.createElement('i');
-  ellipsis.setAttribute('class', 'fa fa-ellipsis-v');
-  list.appendChild(ellipsis);
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  addList(todoTask.value, false, localGet().length + 1);
+  displayList();
```

Comment:
- [ ] This is probably the cause of the add functionality not working fine. You are listening for a submit event on a button. It would be better if you would use the `click` event or include the button inside a form and add the event listener to the form instead.